docs/changelog-fragments/280.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Alter channel.exec_command to always use a newly created ssh_channel to avoid
+segfaults on repeated calls -- :user:`Qalthos`
```

Comment:
Let's use the past tense per the writing style recommendations outlined @ https://github.com/ansible/pylibssh/tree/devel/docs/changelog-fragments#alright-so-how-do-i-add-a-news-fragment
```suggestion
Improved ``channel.exec_command`` to always use a newly created ``ssh_channel`` to avoid
segfaults on repeated calls -- by :user:`Qalthos`
```